src/sidebars/favourites/tearout-directive.js

```diff
@@ -160,6 +161,10 @@
                         function boundsChangingEvent() {
                             if (insideFavouritesPane()) {
                                 reorderFavourites();
+                                dragService.setOtherInstance(null);
```

Comment:
Yeah, I know what you mean, I'll add a shortcut function. To be honest, I think there's only one time it needs to be called. I'll try reducing the number of times I call it (so it's just before the service is destroyed)